Sage/carbon
# docs: use official GA plugin FE-2515

### Proposed behaviour
Use the [official google-analytics addon](https://github.com/storybookjs/storybook/tree/v5.3.3/addons/google-analytics).

### Current behaviour
We have a modified version of the official plugin while we were waiting for [a bug](https://github.com/storybookjs/storybook/issues/6012) to be fixed.




### Checklist


<del>- [ ] Unit tests
<del>- [ ] Cypress automation tests
<del>- [ ] Theme support
<del>- [ ] Typescript `d.ts` file added or updated

### Additional context

https://github.com/storybookjs/storybook/issues/6012
FE-2515

### Testing instructions
- You will need to run `npm install`.
- `npm run storybook` should run without error.
- You will see `https://www.google-analytics.com/analytics.js` being loaded in the network panel.
